README.md

```diff
@@ -18,3 +18,11 @@ You may also find my companion graph [keep-pricefeed](https://github.com/miracle
 # Development
 
 - Run ./update-yamls.js to regenerate the subgraph.yaml files for both mainnet and ropsten based on the template.
+
+# Development Ropsten (Auto)
+- Run `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=<key> ETH_RPC_URL=<url> SUBGRAPH_DEPLOY_KEY=<slug> GRAPH_DEPLOY_KEY=<key> ./deploy-subgraph.sh`
+
+  - `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY`: for HDWalletProvider
```

Comment:
This is probably not necessary, please see the comment in `update-ropsten-subgraph.js`.

Reply:
4a6cb93